lib/test-run.js

```diff
@@ -51,8 +53,8 @@ function * display (pipeline, number, { heroku }) {
 }
 
 function * displayAndExit (pipeline, number, { heroku }) {
-  let testRun = yield display(pipeline, number, { heroku })
-  process.exit(testRun.exit_code)
+  let testNode = yield display(pipeline, number, { heroku })
```

Comment:
`display` is still returning the `testRun`. I guess we should change that to return `firstTestNode`?